compare_platforms/build/Makefile.review

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 The CapableVMs "CHERI Examples" Contributors.
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+# Common Makefile for the privesc examples.
```

Comment:
I believe this is not the "privesc" example. 

I'm unsure what purpose this Makefile has. I'm used to the convention of having "Makefile.<platform>" to build for target <platform>, but this seems to refer to a header file?

Reply:
I followed the structure of the original example that I based this one on. As we discussed, it would be better to clean up the makefiles so that all examples can be built with the top-level makefiles. 

For now, I've corrected the comment from "privesc" to "compare_platforms".

Reply:
Untracked and ignored patch and .DS_Store.

Reply:
> I believe this is not the "privesc" example.
> 
Fixed in 84d32d463865d0ff38a349e845ffbed0ea7cd48a